File: src/CustomSDK/05_Examples/CPlusPlus/CustomModules/CustomModule/src/CustomModule.cpp
#include "stdafx.h"

#include <iostream>

#include "CustomModule/CustomModule.h"

CustomModule::CustomModule() :
  AbstractCustomModule()
{
}

CustomModule::~CustomModule()
{
}

void CustomModule::doSayHello()
{
  std::cout << "[CPlusPlus]:CustomModule::Hello World!" << std::endl;
}

CUSTOMMODULE_C_WRAPPER(CustomModule);

File: src/CustomSDK/05_Examples/Java/src/Examples/CustomModules/CustomModule4Java/src/CustomModule4Java/CustomModule.java
package CustomModule4Java;

import CustomFramework4Java.AbstractCustomModule;
import CustomFramework4Java.LogManager;

public class CustomModule extends AbstractCustomModule
{
  public CustomModule()
  {
    super();
  }

  @Override
  public void doSayHello()
  {
    System.out.println("[Java]:CustomModule::Hello World!");
  }

  public static void doRegister()
  {
    System.out.println("[Java]:"+ CustomModule.class.getName() +" Registered...");
  }
}
